scipy/signal/C_bspline_util.c

```diff
@@ -261,7 +261,7 @@ C_separable_2Dconvolve_mirror(__complex__ float *in, __complex__ float *out,
 	    outptr += N;
 	}
     }
-    else
+    else if(inptr != NULL)
 	memmove(tmpmem, inptr, M*N*sizeof(__complex__ float));
```

Comment:
I think this is a typo and should be `in` not `inptr`. `inptr` shouldn't even be defined in this scope.

Reply:
Thanks!

Reply:
Good catch!

These lines have the same problem it looks like:
```C
memmove(outptr, tmpmem, M*N*sizeof(__complex__ float));
```
should be `out` rather than `outptr`.

Reply:
Also this assumes both arrays are contiguous, but the other `if` case is respecting strides. Clearly this code needs more testing.